bundles/binding/org.openhab.binding.mqttitude/src/main/java/org/openhab/binding/mqttitude/internal/MqttitudeBinding.java

```diff
@@ -118,6 +120,15 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
 			logger.debug("Mqttitude binding configuration updated, 'home' location specified ({}) with a geofence of {}m.", homeLocation.toString(), geoFence);
         }
 		
+		maxGpsAccuracy = Integer.parseInt(getOptionalProperty(properties, "max_gps_accuracy", "-1"));
+		if (maxGpsAccuracy < 0)
+		{
+			logger.debug("Mqttitude binding configuration: no limit on GPS accuracy");
+		}
+		else {
+			logger.debug("Mqttitude binding configuration: GPS accuracy limit is {}", maxGpsAccuracy);
+		}
+		
```

Comment:
> However, are we requiring ESH formatting for openHAB 1 code? If so, where is the formatter configuration in the openhab repository? I've searched for it but haven't found it, at least in the version 1.8 branch where this code lives.

If you use the new IDE, then it enforces it - I guess people using older IDEs don’t have the ESH formatter installed...